doc/security/process.rst

```diff
@@ -7,9 +7,9 @@ Vulnerability Management Process
    surrounding the reported issue.
 #. If the team does not confirm the report, no further action will be
    taken and the issue will be closed.
-#. If the team confirms the report, a unique CVE identifier will be
-   assigned and shared with the reporter. The team will take action to
-   fix the issue.
+#. If the Ceph security team confirms the report, a unique CVE identifier will
+   be assigned to the report and then shared with the reporter. The Ceph
+   security team will start working on a fix. 
```

Comment:
it might not be the 'security team' itself who fixes an issue - perhaps why the language was vague - but the ceph developer team in general

Reply:
Roger that. I've changed the text so that it's vaguer (but syntactically tighter).